app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -702,15 +785,15 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
                                     }
                                 } else {
                                     updateDownloadFailure()
-                                    Utilities.showToastInMidLayout(activity as Context, getString(R.string.settings_local_blocklist_desc4), Toast.LENGTH_SHORT)
+                                    Utilities.showToastUiCentered(activity as Context, getString(R.string.settings_local_blocklist_desc4), Toast.LENGTH_SHORT)
                                 }
                             }
                         }
                     }
                     response.body!!.close()
                     client.connectionPool.evictAll()
                 } catch (e: java.lang.Exception) {
-                    Log.w(LOG_TAG, "$FILETAG Exception while downloading: ${e.message}", e)
+                    Log.w(LOG_TAG_DOWNLOAD, "$FILETAG Exception while downloading: ${e.message}", e)
                     activity?.runOnUiThread {
                         updateDownloadFailure()
```

Comment:
move `runOnUiThread` within the methods, because it is oft repeated?

Reply:
Moved `runOnUiThread` within the method.